src/Faker/Provider/Address.php

```diff
@@ -104,20 +104,24 @@ public static function country()
     }
 
     /**
-     * @example 77.147489
+     * @example '77.147489'
+     * @param float|int $min
+     * @param float|int $max
      * @return float Uses signed degrees format (returns a float number between -90 and 90)
      */
-    public static function latitude()
+    public static function latitude($min = -90, $max = 90)
     {
-        return static::randomFloat(6, 0, 180) - 90;
+        return floatval(number_format(mt_rand($min * 1000000, $max * 1000000)/1000000, 6));
```

Comment:
I don't understand the interest of converting a float to string (with `number_format`) and then back to float (with `floatval`)